zerver/openapi/zulip.yaml

```diff
@@ -4983,7 +4983,7 @@ paths:
   /user_uploads/{realm_id_str}/{filename}:
     get:
       operationId: get_file_temporary_url
-      summary: NO MD FILE.
+      summary: Get file's temporary URL
```

Comment:
Possibly just `Get public temporary URL` in an "Files" section makes the most sense.